birdhouse/optional-components/scheduler-job-deploy_raven_testdata/default.env

```diff
@@ -1,29 +1,20 @@
+export SCHEDULER_JOB_RAVEN_DEPLOY_DATA_JOB_NAME=deploy_raven_testdata_to_thredds
+export SCHEDULER_JOB_RAVEN_DEPLOY_DATA_JOB_COMMENT="Auto-deploy Raven testdata to Thredds for Raven tutorial notebooks."
 export SCHEDULER_JOB_RAVEN_DEPLOY_DATA_JOB_CHECKOUT_CACHE='${BIRDHOUSE_DATA_PERSIST_ROOT}/deploy_data_cache/deploy_raven_testdata_to_thredds'
 export SCHEDULER_JOB_RAVEN_DEPLOY_DATA_JOB_LOG_FILENAME='deploy_raven_testdata_to_thredds.log'
```

Comment:
JOB_LOG_FILENAME also have default generated from JOB_NAME, see https://github.com/bird-house/birdhouse-deploy/blob/b5cd7f6501793ea8e67f76d2208e02dc797ba031/birdhouse/components/scheduler/deploy_data_job.env#L45-L48

The previous `deploy_raven_testdata_to_thredds.env`, only 4 vars need to be set, the rest have generated defaults.  This `default.env` should be similar.  See https://github.com/bird-house/birdhouse-deploy/blob/3a1fa930d7ca329f76ea12696ba5c9146f933b94/birdhouse/components/scheduler/deploy_raven_testdata_to_thredds.env#L1-L7